app/models/concerns/can_change_registration_status.rb

```diff
@@ -70,4 +77,32 @@ def log_status_change
       logger.debug "Changing from #{aasm.from_state} to #{aasm.to_state} (event: #{aasm.current_event})"
     end
   end
+
+  private
+
+  # expires_on is stored as a Time in UTC and then converted to a Date.
+  # If a user first registered near midnight around the transition between GMT and BST (or the other way round),
+  # there is a risk that the UTC date will not be the same as the UK date.
+  # So we should check for this and compensate to avoid expiring them on the wrong date.
+  def expiry_time_adjusted_for_daylight_savings
+    # If registered in BST and expiring in GMT, add an hour
+    if registered_in_daylight_savings? && !expires_in_daylight_savings?
```

Comment:
It's a minor quibble, but I'd rather see an expressive method name than `if true && !true` expression.

How would you feel about creating a method that expresses what this statement is trying to say e.g. `registered_in_bst_and_expiring_in_gmt?`.

Same goes for the `elsif` below.

Reply:
Nope, that totally makes sense! I'll pop that in.